### ISSUE TITLE
Changing ToC orientation only changes one language

### ISSUE THREAD

**Issue by spencerwahl** [ramp4-pcar4/storylines-editor#448]
## Description

If you change the ToC orientation after creating a storylines, the choice is only reflected in the config matching the editor's language.

## Steps to reproduce

1. Create new storyline
2. Select `horizontal` ToC orientation
3. Proceed to main editor
4. Click on edit metadata and switch to `vertical` ToC
5. Return to main editor
6. Open english preview, see vertical ToC
7. Switch to french with top right button, see horizontal ToC.

## Expected behavior

The french config should also get changed

**Comment by szczz** [comment]
I agree that this should be the expected behaviour as we define the ToC orientation in the storyline, but is there a scenario where a storyline might need a vertical ToC in one config and horizontal in the other? 

Happy bug for a future use case? Should be fixed for now
